schemas/blocks/question.json

```diff
@@ -10,6 +10,9 @@
       "number": {
         "type": "string"
       },
+      "cancel_text": {
+        "type": "string"
+      },
```

Comment:
This would technically mean all blocks would support `cancel_text` and not just the `add_block`. Not sure if it worth making an add_block question type just for it.

Reply:
I'm not sure why it should just be on the add block. Maybe it should be on add, change and remove?

Reply:
I've added a new question type for list add/edit/remove questions.